helpers/hyper-v-mutating-webhook/webhook.go

```diff
@@ -1,69 +1,73 @@
-// based on https://github.com/kubernetes-sigs/controller-runtime/tree/master/examples/builtins
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package main
 
 import (
 	"context"
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"strings"
 
 	corev1 "k8s.io/api/core/v1"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-// +kubebuilder:webhook:path=/mutate-v1-pod,mutating=true,failurePolicy=fail,groups="",resources=pods,verbs=create;update,versions=v1,name=mpod.kb.io
+// +kubebuilder:webhook:path=/mutate-v1-pod,mutating=true,failurePolicy=fail,groups="",resources=pods,verbs=create;update,versions=v1,name=mpod.kb.io,admissionReviewVersions={v1},sideEffects=None
 
 var (
-	webhookLogger        = ctrl.Log.WithName("webhook")
-	windows2019          = "windows-2019"
-	windows2004          = "windows-2004"
-	windows2004tagSuffix = "-windows-amd64-2004"
```

Comment:
We no longer need to expand out the tag nane?

Reply:
These are no longer needed because the test scenarios we are targeting only support running the same WS version for the container and node.